server/cluster.go

```diff
@@ -206,6 +220,10 @@ func makeStoreKeyPrefix(clusterRootPath string) string {
 	return strings.Join([]string{clusterRootPath, "s", ""}, "/")
 }
 
+func makeBootstrapTimeKey(clusterRootPath string) string {
+	return strings.Join([]string{clusterRootPath, "bootstrap_time"}, "/")
```

Comment:
the cluster root path already include "raft" https://github.com/pingcap/pd/pull/645/files/feb470d10155b16c2144b964c8becb920e9c64f7#diff-d412b9240514b30a308e4b4c8208ae72R169